sa/SASCM.CLEO.ini

```diff
@@ -130,4 +130,18 @@
 0B1A=1,~%1d%
 0B1B=2,%1d% %= %2d%
 0B1C=2,%1d% >>= %2d%
-0B1D=2,%1d% <<= %2d%
\ No newline at end of file
+0B1D=2,%1d% <<= %2d%
+
+; CLEO 5 Opcodes
+
+0DD5=1,%1d% = get_platform ; PC
+2000=2,%2s% = resolve_filepath %1s% 
+2001=3,%3s% = get_script_filename %1s% full_path %2d%
+
+; CLEO 5 DebugUtils
+
+00C3=0,debug_on
+00C4=0,debug_off
+00CC=-1,breakpoint
+00CD=-1,trace %1s%
+00CE=-1, log file %1s% timestamp %2d% msg %3s%
```

Comment:
log_file ? I think log is already taken by math logarithm.

Reply:
just copied from your example) can update to log_to_file